opentreemap/treemap/js/src/lib/geocoderUi.js

```diff
@@ -0,0 +1,64 @@
+"use strict";
```

Comment:
I confirmed that the bad address searches are fixed.

I am still able reproduce the "select boundary with down arrow" problem. I am seeing slightly different behavior now. It correctly selects and zooms to the 3rd region, but then immediately selects and zooms to the first region. The back button history has both regions.  I captured a GIF of it.

![otm-region-arrow-key-select-behavior](https://cloud.githubusercontent.com/assets/17363/24169980/86ad3f40-0e3c-11e7-9f71-4b63cf8b237c.gif)


Possibly related: if I use the mouse to select a region from the autocomplete list, on staging, I get the solid color overlay

![screen shot 2017-03-21 at 1 42 02 pm](https://cloud.githubusercontent.com/assets/17363/24169950/60bcbd56-0e3c-11e7-923b-1a7837545d62.png)

On this branch I do not

![screen shot 2017-03-21 at 1 42 22 pm](https://cloud.githubusercontent.com/assets/17363/24169975/77b9a97e-0e3c-11e7-862b-8d33b05e6108.png)